distribution/feature-packs/server-feature-pack/src/main/resources/licenses/keycloak/Emotion,MIT License.txt

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
JSON in txt file

Reply:
You can probably use https://raw.githubusercontent.com/emotion-js/emotion/master/LICENSE